Configuration/CommandLineOption.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+
+namespace QuantConnect.Configuration
+{
+    /// <summary>
+    /// Auxiliary class to keep information about a specific command line option
+    /// </summary>
+    public class CommandLineOption
+    {
+        /// <summary>
+        /// Command line option contructor
+        /// </summary>
+        public CommandLineOption(string name, CommandOptionType type, string description = "")
+        {
+            Type = type;
+            Description = description;
+            Name = name;
+        }
+
+        /// <summary>
+        /// Command option type
+        /// </summary>
+        public CommandOptionType Type { get; }
+
+        /// <summary>
+        /// Command option description
+        /// </summary>
+        public string Description { get; }
```

Comment:
no change required - General style of the project places properties at the top. I *think* it's generally like this:

private mutable state
private state
(generally should *never* have public fields, but they do occur sometimes in the code-base)
public properties
ctors (ordered by accessibility)
public methods
abstract methods
protected methods
private methods

This will be included in a R# `dotSettings` file *one-day* when I get some free time

Reply:
I'll change it ;)